gulpfile.js

```diff
@@ -20,11 +20,7 @@ const through = require('through');
 
 	const SCRIPTS = {
 		main: {source: 'src/index.js', target: 'build/potree/potree.js', args: {standalone: 'Potree'}},
-		bin: {source: 'src/workers/BinaryDecoderWorker.js', target: 'build/potree/workers/BinaryDecoderWorker.js'},
-		dem: {source: 'src/workers/DEMWorker.js', target: 'build/potree/workers/DEMWorker.js'},
-		gre: {source: 'src/workers/GreyhoundBinaryDecoderWorker.js', target: 'build/potree/workers/GreyhoundBinaryDecoderWorker.js'},
-		las: {source: 'src/workers/LASDecoderWorker.js', target: 'build/potree/workers/LASDecoderWorker.js'},
```

Comment:
The LasDecoder is very big (1MB or so), would be good if - at least - that one would stay its own worker.

Reply:
so that actually might be because of crypto erroneously getting bundled into that one, let me pull that part out

Reply:
the las decoder worker is only 2.5k its the greyhound and laz loader that are large, ~700k with the crypto fix, but the actually are relying on the same library so combining those 2 would save the user from having to download laz-perf twice, though I realized I didn't include laz loader in this pull